chainer/gradient_check.py

```diff
@@ -812,6 +838,17 @@ def func(xs):
     ).run()
 
 
+def _check_backward_with_params(
```

Comment:
This function needs some comment: what's the difference, why it's needed, who's using it, etc.

Reply:
Fixed.

Reply:
Fixed.